### DISCUSSION TITLE
Is it possible to use gapseq to analyze tens or hundreds of genomes simultaneously?

### DISCUSSION THREAD

**Discussion by Dandlion-Z** [jotech/gapseq#222] [Q&A]
Is it possible to use the gapseq code published on GitHub to **analyze dozens or hundreds of Genomes simultaneously**, so that there is no need to use gapseq to analyze each Genome separately？

**Comment by Waschina** [comment]
Duplicate to https://github.com/jotech/gapseq/issues/52

You can start parallel instances of gapseq. There are several ways to do that for command line tools; one option is mentioned in the abovementioned issue.

**Reply by Dandlion-Z** [reply]
Thank you very much for your reply, and we will continue to try to solve the problem according to your suggestions.

**Comment by Dandlion-Z** [comment]
Thanks for your reply! Your approach is really helpful with our questions.


992461451
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jotech/gapseq"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月5日(星期五) 晚上6:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jotech/gapseq] Is it possible to use gapseq to analyze tens or hundreds of genomes simultaneously? (Discussion #222)





 
Duplicate to #52
 
You can start parallel instances of gapseq. There are several ways to do that for command line tools; one option is mentioned in the abovementioned issue.
